tests/systemtests/cli.go

```diff
@@ -169,6 +169,16 @@ func (c CLIWrapper) Run(args ...string) string {
 	return rsp
 }
 
+// RunCommandWithArgs use for run cli command, not tx
+func (c CLIWrapper) RunCommandWithArgs(args ...string) string {
+	c.t.Helper()
+	execOutput, ok := c.run(args)
```

Comment:
It always returns the execOutput here, shouldn't it return an error if not okay? Or is that expected that ok is ignored?

Reply:
I think should ignore it with the local command. `execOutput` already returns err if cmd failed, handler by `run` func.